Base room, plus other stuff

Juhoton opened the pull request:
FIrst version of the Base room. Made in a new scene, and a made it into a new prefab, so that I can continue the editing of the room, and it should make those changes to other people using the room. Also changed the location of wall related items to the building folder. Changed some wall names.

viilis commented on Assets/Materials/Television_var.mat:
```diff
@@ -26,5 +26,5 @@ Material:
     m_Floats: []
     m_Colors:
-    - _Color: {r: 0.3058179, g: 0.3058179, b: 0.3058179, a: 0}
+    - _Color: {r: 0.30581748, g: 0.30581748, b: 0.30581748, a: 0}
```
unnecessary change

Juhoton replied:
Yeah, I don't actually know why that was changed. If I remember correctly, I didn't touch the asset.

viilis commented on Assets/Models/Building/RightCornerWall4 - copy.fbx:
Why there are copies of models?

Juhoton replied:
Found similar models, so named the extra ones into copies, rather than deleting them

viilis commented on Assets/Models/Building/WallAtlas - Copy.png:
same here

viilis commented on Assets/Models/Building/Wall4 - copy.fbx:
copy here

viilis commented on Assets/Models/Building/LeftCornerWall3 - copy.fbx:
copy?

Juhoton commented:
Removed the copies. Changed the movable_player camera height to 0.6 from 0.87. So currently it should be around 1.6 meters, which is closer to where the average eyes actually are. Also dropped the fov to 70, have to check around what feels best and creates better immersion in game. Television_var is fucked.
